compiler/damlc/daml-ide-core/src/Development/IDE/Core/Rules/Daml.hs

```diff
@@ -474,10 +474,12 @@ generateDocTestModuleRule =
 generatePackageMap :: LF.Version -> Maybe NormalizedFilePath -> [FilePath] -> IO ([FileDiagnostic], Map.Map UnitId LF.DalfPackage)
 generatePackageMap version mbProjRoot userPkgDbs = do
     versionedPackageDbs <- getPackageDbs version mbProjRoot userPkgDbs
+    depDbs' <- getDependenciesDbs version mbProjRoot
```

Comment:
I tried without and the build failed, so looks like we still depend on it. Also, I think it would be safer to include it for now so that projects with an already built package-db won't break.

Reply:
We need to reinitialize the package db on sdk upgrades anyway so I don’t think existing projects matter much. I’m a bit worried that if we keep this we end up with a somewhat inconsistent state where it’s unclear what ends up where but as long as we get rid of it in follow-up PRs I’m happy.

Reply:
So I managed to remove the `projectPackageDatabase`. We still depend on the `userPkgDbs` for the bootstrapping process of damlc.